// Copyright (C) 2016 Nippon Telegraph and Telephone Corporation.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//    http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
// implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";

package knetservice;

import "knet.proto";

service KNET {
    rpc InitKNET(KNET.InitRequest) returns (KNET.InitResponse) {}
    rpc AddKNET(KNET.AddRequest) returns (KNET.AddResponse) {}
    rpc DeleteKNET(l2.DeleteAddressRequest) returns (KNET.DeleteResponse) {}
    rpc ListKNET(KNET.ListRequest) returns (KNET.ListResponse) {}
}